build/csslint/xfail.list

```diff
@@ -53,6 +53,7 @@ apps/settings/style/downloads.css 0 3
 apps/sms/style/root.css 0 1
 apps/sms/style/sms.css 0 4
 apps/system/mobile_id/style/mobile_id.css 0 16
+apps/system/style/devtools/devtools_auth.css 1 0
```

Comment:
@Rik ah, perhaps you are right:
```
Unknown property 'object-fit'.  Declaration dropped. line: 31 column: 12 source: "  object-fit: cover;"
```
I suppose I'll just skip the error for now then.